core/station/api/spec.did

```diff
@@ -1177,6 +1183,12 @@ type ListRequestsResult = variant {
 type GetRequestInput = record {
   // The request id to retrieve.
   request_id : UUID;
+  // Fill in all the additional info about the request operation, request types such as Call External Canisters will
+  // include the request argument and reply blobs, this can be a large amount of data and could potentially exceed the
+  // response size limit.
+  //
+  // By default, this field is set to `true`.
```

Comment:
What does "by default" mean? That none is unwrapped to true?

Reply:
> this can be a large amount of data and could potentially exceed

shouldn't the default be false to avoid exceeding the limits?

Reply:
> What does "by default" mean? That none is unwrapped to true?

Correct, but actually i'll move this to `false` since this will be less risky for clients and if they set to true they will need to explicitely handle a possible fail condition.